src/docs/components/JSXGenerator.js

```diff
@@ -30,13 +29,6 @@ function generateJSX(component) {
   let jsxProps = Object.keys(component.props)
     .filter(key => key !== 'children')
     .filter(key => component.props[key] !== undefined)
-    .filter(key => {
-      const componentWithoutProp = React.cloneElement(component, {[`${key}`]: undefined});
-      const codeWithProp = ReactDOMServer.renderToStaticMarkup(component);
-      const codeWithoutProp = ReactDOMServer.renderToStaticMarkup(componentWithoutProp);
-
-      return codeWithProp !== codeWithoutProp;
-    })
```

Comment:
AFAIK this was moved to `DocsActiveBlock.removePropsWithDefaultValues`

Reply:
not all of it - we still need to compare html to not generate jsx for default values, like normal size in Label

Reply:
good point, thanks for catching that!